rules.txt

```diff
@@ -1,12 +1,215 @@
-Submission Rules:
-  All submissions must NOT be:
-    Questions / requests for support.
-    NSFW.
-    Memes.
-    Links to sites that require logins or have a paywall.
-  All submissions MUST be:
-    The original article - no link shorteners OR spamblogs, please!
-    Compliant with the site-wide reddit rules.
-Comment Rules:
-  All comments MUST be:
-    Compliant with the site-wide reddit rules.
+#Submission Rules:
+#  All submissions must NOT be:
+#    Questions / requests for support.
+#    NSFW.
+#    Memes.
+#    Links to sites that require logins or have a paywall.
+#  All submissions MUST be:
+#    The original article - no link shorteners OR spamblogs, please!
+#    Compliant with the site-wide reddit rules.
+#Comment Rules:
+#  All comments MUST be:
+#    Compliant with the site-wide reddit rules.
+---
+##----------------------------
+##    Affiliate Links Rules
+##----------------------------
+#TODO
+---
+##----------------------------
+##    Click-bait Rules
+##----------------------------
+#TODO
+---
+##----------------------------
+##    Disguised Links Rules
+##----------------------------
+#TODO
+---
+##----------------------------
+## Language/Profanity - (mostly from automoderator example)
+##----------------------------
+    title+body (regex): ["a(ss|rse|es)([ -]?holes?)?","b(i|\\*)o?(tch|\\*{3})(y|es)?","cocks?([ -]?suck(ers?|ing)?)?","cum(ming|[ -]shots?)?","cunts?","((mother|motha|mutha)[ -]?)?f(u?c?k?k|\\*ck|\\*{0,2}k|\\*{3})(er|ed|ing|s)?", "s(h(i|ar?|\\*)t|\\*{3}|h\\*{2})(s|ter|e|ting)?", "n[ioe0@][g]{1,}[aioe0@]{1,}r", "f[aioe0@]{1,}[g]{1,}[aioe03@]{1,}t"]
```

Comment:
Profanity shouldn't be an instant remove, or a remove at all in my opinion

Reply:
I agree that not all profanity should be instant removed and would like to move some of them to a report-only section. Some of the others should remain as instant removal, like the N word, though.